Sources/Sentry/SentryNetworkTrackingIntegration.m

```diff
@@ -74,36 +73,6 @@ + (void)swizzleURLSessionTask
     }
 }
 
-+ (void)swizzleNSURLSessionConfiguration
```

Comment:
`h`: Why don't we need this anymore? Fantastic, if we can get rid of some swizzling.

Reply:
Because the only thing that we did in there (adding the baggage header) we no longer do :)